uri.go

```diff
@@ -101,3 +103,21 @@ type URIWithIcon interface {
 
 	Icon() Resource
 }
+
+// OpenURI asks the operating system to open the specified resource.
+// This is commonly used for a URL or a file URI to open the appropriate system application.
+//
+// Since: 2.6
+func OpenURI(u URI) error {
+	a := CurrentApp()
+	if a == nil {
+		return errors.New("oepnURI requires a running Fyne application")
```

Comment:
Fixing spelling mistake and prefix with a word so the name can be spelled with a capital case in the beginning:
```suggestion
		return errors.New("running OpenURI requires a running Fyne application")
```

Reply:
Good catch. I used a different word to avoid the duplication.

Reply:
Thanks. I'd still recommend going with the same function name though. It seems like you forgot to use a capital letter there.